### PR TITLE
Add link to item tracker on the item tracker facade site

### PR DESCRIPTION
**Pull request by neptuwunium**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Website stuff
 
-Nothing to see here
\ No newline at end of file
+Because I am both Lazy and have bad foresight, the item tracker is actually [here](https://js41637.github.io/Overwatch-Item-Tracker/).
```
Comment (Js41637) [inline readme.md]: why is lazy capitalized here?

**Review by Js41637 — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (neptuwunium) [reply readme.md]: to be ironic

**Comment by ZingBallyhoo** [comment]
thanks for this feature, i really needed it

**Comment by Js41637** [comment]
ur welcome

**Pull request merged**